package.json

```diff
@@ -2,14 +2,24 @@
   "name": "classnames",
   "version": "2.3.1",
   "description": "A simple utility for conditionally joining classNames together",
+  "type": "module",
+  "sideEffects": false,
   "main": "index.js",
+  "exports": {
+    ".": "./index.js",
+    "./bind": "./bind.js",
+    "./dedupe": "./dedupe.js"
```

Comment:
I'm following the examples from the [Node documentation](https://nodejs.org/api/packages.html#packages_subpath_exports) which seem to prefer the style without extensions. I am inclined to follow the extension-less style as it matches the existing API surface and would not require users to update their own code.

Unless I am mistaken tools like UNPKG will still properly resolve the correct filename (such as http://unpkg.com/classnames@latest/dedupe) since we are not changing the root structure of the package. Meaning that since we have `dedupe.js` in the root of the our file structure it can still be resolved by tools that do not support the `exports` field.

There are also alternative tools such as Skypack that are [actively encouraging](https://www.skypack.dev/blog/2020/10/skypack-quality-score-actionable-feedback-to-build-better-packages/) including these fields as well, extensions or not.